### PR TITLE
MM-31513 Document how team admins can see their teams' statistics

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/docs/issues/4141

Updated:
- Self-Managed Admin Guide > Administration > Statistics
   - Updated the "Troubleshooting" subsection title to include "FAQ" as "Troubleshooting/FAQ"
   - Formatted the existing troubleshooting tip as a Q&A
   - Added another Q&A section to detail how to use system admin roles to provide Team Admins with access to team stats

### CODE REVIEW

File: source/administration/statistics.rst [hunk 1]
```diff
@@ -102,7 +102,27 @@ Recent Active Users
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 If the statistics page is loading endlessly and you get an error message saying "Not enough data for a meaningful representation", check whether you're using an ad blocker. An ad blocker can prevent this page from loading data. To test this, temporarily disable your ad blocker, or view the page in a browser without an ad blocker installed.
+
+Can Team Admins review their own team's statistics?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Available in Enterprise Edition E20
+
+System Administrators can designate the **Viewer** `System Admin Role <https://docs.mattermost.com/deployment/admin-roles.html>`__ to enable Team Administrators  to see team statistics.
```
Comment (cwarnermm) [inline source/administration/statistics.rst:118]:
```suggestion
System Administrators can designate the **Viewer** `System Admin Role <https://docs.mattermost.com/deployment/admin-roles.html>`__ to enable Team Administrators to see team statistics.
```

**Review by wiersgallak — APPROVED** [review]

File: source/administration/statistics.rst [hunk 2]
```diff
@@ -102,7 +102,27 @@ Recent Active Users
...
+
+I see an error: "Not enough data for a meaningful representation"
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 If the statistics page is loading endlessly and you get an error message saying "Not enough data for a meaningful representation", check whether you're using an ad blocker. An ad blocker can prevent this page from loading data. To test this, temporarily disable your ad blocker, or view the page in a browser without an ad blocker installed.
+
+Can Team Admins review their own team's statistics?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Available in Enterprise Edition E20
```
Comment (justinegeffen) [inline source/administration/statistics.rst:116]:
```suggestion
*Available in Enterprise Edition E20.*
```

File: source/administration/statistics.rst [hunk 3]
```diff
@@ -102,7 +102,27 @@ Recent Active Users
...
+
+System Administrators can designate the **Viewer** `System Admin Role <https://docs.mattermost.com/deployment/admin-roles.html>`__ to enable Team Administrators to see team statistics.
```
Comment (justinegeffen) [inline source/administration/statistics.rst:118]:
```suggestion
System Admins can designate the **Viewer** `System Admin Role <https://docs.mattermost.com/deployment/admin-roles.html>`__ to enable Team Admins to see team statistics.
```

File: source/administration/statistics.rst [hunk 4]
```diff
@@ -102,7 +102,27 @@ Recent Active Users
...
+
+Once Team Admins are assigned to the **Viewer** admin role, they can access all statistics for all teams.
```
Comment (justinegeffen) [inline source/administration/statistics.rst:120]:
```suggestion
Once Team Admins are assigned to the **Viewer** role, they can access all statistics for all teams.
```

File: source/administration/statistics.rst [hunk 5]
```diff
@@ -102,7 +102,27 @@ Recent Active Users
...
+
+To enable Team Admins to see statistics for their teams, System Administrators must then `edit the privileges <https://docs.mattermost.com/deployment/admin-roles.html#editing-privileges-of-admin-roles-advanced>`__ of the **Viewer** admin role to set all privileges to **No access** except the **Reporting** privileges.
```
Comment (justinegeffen) [inline source/administration/statistics.rst:122]:
```suggestion
System Admins must then `edit the privileges <https://docs.mattermost.com/deployment/admin-roles.html#editing-privileges-of-admin-roles-advanced>`__ of the **Viewer** role. Only the **Reporting** privileges need to be enabled to allow Team Admins to see statistics for their teams.
```

File: source/administration/statistics.rst [hunk 6]
```diff
@@ -102,7 +102,27 @@ Recent Active Users
...
+
+Once the Viewer admin role is set up for Reporting access only, System Admins can then `assign <https://docs.mattermost.com/deployment/admin-roles.html#assigning-admin-roles>`__ the **Viewer** admin role to Team Admins.
```
Comment (justinegeffen) [inline source/administration/statistics.rst:124]:
```suggestion
Once the **Viewer** role is set up for reporting access only, System Admins can then `assign <https://docs.mattermost.com/deployment/admin-roles.html#assigning-admin-roles>`__ the **Viewer** role to Team Admins.
```

File: source/administration/statistics.rst [hunk 7]
```diff
@@ -102,7 +102,27 @@ Recent Active Users
...
+
+.. note::
+  - System Admins must manually add people to or remove people from the **Viewer** admin role to address Team Admin changes, such as promotions or demotions.
```
Comment (justinegeffen) [inline source/administration/statistics.rst:127]:
```suggestion
  - System Admins must manually add or remove members from the **Viewer** role to address Team Admin changes, such as promotions or demotions.
```

File: source/administration/statistics.rst [hunk 8]
```diff
@@ -102,7 +102,27 @@ Recent Active Users
...
+
+Once Team Admins are assigned to the **Viewer** admin role, they can access all statistics for all teams.
+
+To enable Team Admins to see statistics for their teams, System Administrators must then `edit the privileges <https://docs.mattermost.com/deployment/admin-roles.html#editing-privileges-of-admin-roles-advanced>`__ of the **Viewer** admin role to set all privileges to **No access** except the **Reporting** privileges.
+
+Once the Viewer admin role is set up for Reporting access only, System Admins can then `assign <https://docs.mattermost.com/deployment/admin-roles.html#assigning-admin-roles>`__ the **Viewer** admin role to Team Admins.
+
+.. note::
+  - System Admins must manually add people to or remove people from the **Viewer** admin role to address Team Admin changes, such as promotions or demotions. 
+  - Team Admins using the **Viewer** admin role will also have access to system level statistics in addition to statistics for their teams.
```
Comment (justinegeffen) [inline source/administration/statistics.rst:128]:
```suggestion
  - Team Admins using the **Viewer** role will also have access to system level statistics in addition to statistics for their teams.
```

File: source/cloud/cloud-reporting.rst [hunk 9]
```diff
@@ -65,7 +65,25 @@ Recent Active Users
...
+
+I see an error: "Not enough data for a meaningful representation"
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 If the statistics page is loading endlessly and you get an error message saying "Not enough data for a meaningful representation", check whether you're using an ad blocker. An ad blocker can prevent this page from loading data. To test this, temporarily disable your ad blocker, or view the page in a browser without an ad blocker installed.
+
+Can Team Admins review their own team's statistics?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+System Administrators can designate the **Viewer** `System Admin Role <https://docs.mattermost.com/cloud/cloud-user-management/user-management.html#about-system-roles-beta>`__ to enable Team Administrators to see team statistics.
```
Comment (justinegeffen) [inline source/cloud/cloud-reporting.rst:79]:
```suggestion
System Admins can designate the **Viewer** `System Admin Role <https://docs.mattermost.com/cloud/cloud-user-management/user-management.html#about-system-roles-beta>`__ to enable Team Admins to see team statistics.
```

File: source/cloud/cloud-reporting.rst [hunk 10]
```diff
@@ -65,7 +65,25 @@ Recent Active Users
...
+
+Once Team Admins are assigned to the **Viewer** admin role, they can access all statistics for all teams.
```
Comment (justinegeffen) [inline source/cloud/cloud-reporting.rst:81]:
```suggestion
Once Team Admins are assigned to the **Viewer** role, they can access all statistics for all teams.
```

File: source/cloud/cloud-reporting.rst [hunk 11]
```diff
@@ -65,7 +65,25 @@ Recent Active Users
...
+
+To enable Team Admins to see statistics for their teams, System Administrators must then `edit the privileges <https://docs.mattermost.com/cloud/cloud-user-management/user-management.html#editing-privileges-of-system-roles>`__ of the **Viewer** admin role to set all privileges to **No access** except the **Reporting** privileges.
+
+Once the Viewer admin role is set up for Reporting access only, System Admins can then `assign <https://docs.mattermost.com/cloud/cloud-user-management/user-management.html#assigning-system-roles>`__ the **Viewer** admin role to Team Admins.
```
Comment (justinegeffen) [inline source/cloud/cloud-reporting.rst:85]:
```suggestion
Once the **Viewer** role is set up for reporting access only, System Admins can then `assign <https://docs.mattermost.com/deployment/admin-roles.html#assigning-admin-roles>`__ the **Viewer** role to Team Admins.
```

File: source/cloud/cloud-reporting.rst [hunk 12]
```diff
@@ -65,7 +65,25 @@ Recent Active Users
...
 If the statistics page is loading endlessly and you get an error message saying "Not enough data for a meaningful representation", check whether you're using an ad blocker. An ad blocker can prevent this page from loading data. To test this, temporarily disable your ad blocker, or view the page in a browser without an ad blocker installed.
+
+Can Team Admins review their own team's statistics?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+System Administrators can designate the **Viewer** `System Admin Role <https://docs.mattermost.com/cloud/cloud-user-management/user-management.html#about-system-roles-beta>`__ to enable Team Administrators to see team statistics.
+
+Once Team Admins are assigned to the **Viewer** admin role, they can access all statistics for all teams.
+
+To enable Team Admins to see statistics for their teams, System Administrators must then `edit the privileges <https://docs.mattermost.com/cloud/cloud-user-management/user-management.html#editing-privileges-of-system-roles>`__ of the **Viewer** admin role to set all privileges to **No access** except the **Reporting** privileges.
```
Comment (justinegeffen) [inline source/cloud/cloud-reporting.rst:83]:
```suggestion
System Admins must then `edit the privileges <https://docs.mattermost.com/deployment/admin-roles.html#editing-privileges-of-admin-roles-advanced>`__ of the **Viewer** role. Only the **Reporting** privileges need to be enabled to allow Team Admins to see statistics for their teams.
```

File: source/cloud/cloud-reporting.rst [hunk 13]
```diff
@@ -65,7 +65,25 @@ Recent Active Users
...
+
+Once the Viewer admin role is set up for Reporting access only, System Admins can then `assign <https://docs.mattermost.com/cloud/cloud-user-management/user-management.html#assigning-system-roles>`__ the **Viewer** admin role to Team Admins.
+
+.. note::
+  - System Admins must manually add people to or remove people from the **Viewer** admin role to address Team Admin changes, such as promotions or demotions.
```
Comment (justinegeffen) [inline source/cloud/cloud-reporting.rst:88]:
```suggestion
  - System Admins must manually add or remove members from the **Viewer** role to address Team Admin changes, such as promotions or demotions.
```

File: source/cloud/cloud-reporting.rst [hunk 14]
```diff
@@ -65,7 +65,25 @@ Recent Active Users
...
+
+Once Team Admins are assigned to the **Viewer** admin role, they can access all statistics for all teams.
+
+To enable Team Admins to see statistics for their teams, System Administrators must then `edit the privileges <https://docs.mattermost.com/cloud/cloud-user-management/user-management.html#editing-privileges-of-system-roles>`__ of the **Viewer** admin role to set all privileges to **No access** except the **Reporting** privileges.
+
+Once the Viewer admin role is set up for Reporting access only, System Admins can then `assign <https://docs.mattermost.com/cloud/cloud-user-management/user-management.html#assigning-system-roles>`__ the **Viewer** admin role to Team Admins.
+
+.. note::
+  - System Admins must manually add people to or remove people from the **Viewer** admin role to address Team Admin changes, such as promotions or demotions. 
+  - Team Admins using the **Viewer** admin role will also have access to system level statistics in addition to statistics for their teams.
```
Comment (justinegeffen) [inline source/cloud/cloud-reporting.rst:89]:
```suggestion
  - Team Admins using the **Viewer** role will also have access to system level statistics in addition to statistics for their teams.
```

**Review by justinegeffen — COMMENTED** [review]
Thanks, @cwarnermm, this is great.

I've suggested the removal of the word "admin" when referring to the role primarily because it was hard for me to figure out whether it should be "admin" or "Admin". We don't generally refer to "Admin" without the preceding Channel/Team/System so this may be a happy compromise?

**Comment by cwarnermm** [comment]
@justinegeffen - Great editor feedback and a very decent compromise for referring to roles versus admin users. Thank you!

**Comment by justinegeffen** [comment]
No problem at all - as long as it doesn't have a huge impact on existing docs? If there is a big impact, I'm happy to help update - just let me know! :)

**Comment by cwarnermm** [comment]
@justinegeffen - What I've seen matches your edits -- we refer to "admin roles" for the topic on System Roles (Beta) itself. I'll keep an eye out for additional references going forward.

**Review by justinegeffen — APPROVED** [review]

**Review by jasonblais — APPROVED** [review]
Overall it's a good start, but as a next iteration for this FAQ, I'd propose making the FAQ more self-contained. For an admin just looking for an answer on "can my Team Admin see the stats for their team, and how do I make that happen", the FAQ feels overwhelming, given the information I need to read and understand on other pages.

Instead, another approach would be to structure the FAQ to be more like:

1. Yes, and here are the 3 steps you need to do to make that happen (go to System Console User Management > System Roles, .. etc)
2. Learn more about System Admin Roles, including how to edit privileges and assign role at <link to more admin roles doc>

**Comment by cwarnermm** [comment]
@jasonblais Thank you for your thoughtful review. I couldn't agree more with your feedback -- it's precisely where we need to take the product documentation to best serve the target audience going forward. I appreciate this feedback because it empowers us to provide better quality documentation to our target audiences.

For this first iteration, I applied a paradigm commonly used in the docs today. The result forces the user to follow a number of threads towards a final goal, which is less than ideal (and a lot of cognitive load for the user to take on themselves.) 

I'll log a ticket in the docs backlog to circle back to complete another iteration of this content that takes your feedback into account. Also, as new docs content is developed, we'll stop applying the existing paradigm of short-cutting details in favour of links, and start providing content that users need most.  CC @justinegeffen

**Comment by justinegeffen** [comment]
@jaydeland, any idea why this is failing the build with error 128 "fatal: Couldn't find remote ref MM-31513-Document-how-team-admins-can-see-their-teams-statistics"?

**Pull request merged**